test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -1074,6 +1090,63 @@ function flux(::Diffusion{en_ρatke}, atmos, args)
     return -gm.ρ * env.a * K_m * en_dif.∇tke[3] * ẑ
 end
 
+function flux(::Diffusion{up_ρa{i}}, atmos, args) where {i}
+    @unpack state, aux = args
+    @unpack ρa_up = args.precomputed.turbconv
+    FT = eltype(atmos)
+    up = state.turbconv.updraft
+    up_dif = diffusive.turbconv.updraft
+    gm = state
+    z = altitute(atmos, aux)
+    α = atmos.turbconv.surface.α_surf
+    K = exp(-α*z)
+    ẑ = vertical_unit_vector(atmos, aux)
+    return -K * up_dif.∇ρa[3] * ẑ
+end
+
+
+function flux(::Diffusion{up_ρaw{i}}, atmos, args) where {i}
+    @unpack state, aux = args
+    @unpack ρa_up = args.precomputed.turbconv
+    FT = eltype(atmos)
+    up = state.turbconv.updraft
+    up_dif = diffusive.turbconv.updraft
+    gm = state
+    z = altitute(atmos, aux)
+    α = atmos.turbconv.surface.α_surf
+    K = exp(-α*z)
```

Comment:
Can you centralize the definition of `K` in some file?

Reply:
Sounds like a good idea. Alternatively, we could add it to `precompute` and unpack it (for `precompute(, ::Flux{SecondOrder})` only).

Reply:
I used a function 
K = surface_upd_diff(α,z)
which is defined in surface_functions.jl